assets/src/components/integrations/IntegrationsOverview.tsx

```diff
@@ -16,16 +29,295 @@ type IntegrationType = {
+    {
+      title: 'Webhook URL',
+      dataIndex: 'webhook_url',
+      key: 'webhook_url',
+      render: (value: string, record: any) => {
+        return (
+          <Text keyboard strong>
+            {value}
+          </Text>
+        );
+      },
+    },
+    {
+      title: 'Status',
+      dataIndex: 'verified',
+      key: 'verified',
+      render: (verified: boolean) => {
+        return verified ? (
+          <Tag color={colors.green}>Verified</Tag>
+        ) : (
+          <Tag>Unverified</Tag>
+        );
+      },
+    },
+    {
+      title: 'Connected since',
+      dataIndex: 'created_at',
+      key: 'created_at',
+      render: (value: string) => {
+        if (!value) {
+          return '--';
+        }
+
+        return dayjs(value).format('MMMM DD, YYYY');
+      },
+    },
+    {
+      title: '',
+      dataIndex: 'action',
+      key: 'action',
+      render: (action: any, record: any) => {
+        return <Button>Update</Button>;
+      },
+    },
+  ];
+
+  return <Table rowKey="id" dataSource={webhooks} columns={columns} />;
+};
+
+// TODO: clean up a bit
+const NewWebhookModal = ({
+  visible,
+  onSuccess,
+  onCancel,
+}: {
+  visible: boolean;
+  onSuccess: (webhook: WebhookEventSubscription) => void;
+  onCancel: () => void;
+}) => {
+  const [url, setWebhookUrl] = React.useState('');
+  const [isVerifying, setIsVerifying] = React.useState(false);
+  const [isVerified, setIsVerified] = React.useState(false);
+  const [isSaving, setIsSaving] = React.useState(false);
+
+  const handleChangeUrl = (e: any) => setWebhookUrl(e.target.value);
+
+  const handleVerifyUrl = async () => {
+    console.log('Verifying:', url);
+    setIsVerifying(true);
+
+    const {verified} = await API.verifyWebhookUrl(url);
+    console.log('Verified?', verified);
+    await sleep(1000);
+
+    setIsVerifying(false);
+    setIsVerified(verified);
+  };
+
+  const handleCancelWebhook = () => {
+    onCancel();
+    setWebhookUrl('');
+    setIsVerified(false);
+  };
+
+  const handleSaveWebhook = async () => {
+    console.log('Saving:', url);
+    setIsSaving(true);
+
+    const webhook = await API.createEventSubscriptions({
+      webhook_url: url,
+    });
+
+    setIsSaving(false);
+    onSuccess(webhook);
+    setWebhookUrl('');
+    setIsVerified(false);
+  };
+
+  return (
+    <Modal
+      title="Add webhook URL"
+      visible={visible}
+      onOk={handleSaveWebhook}
+      onCancel={handleCancelWebhook}
+      footer={[
+        <Button key="cancel" onClick={handleCancelWebhook}>
+          Cancel
+        </Button>,
+        <Button
+          key="submit"
+          type="primary"
+          loading={isSaving}
+          onClick={handleSaveWebhook}
+        >
+          Save
+        </Button>,
+      ]}
+    >
+      <Box>
+        <Paragraph>
+          <Text>
+            You can subscribe to be notified of events in Papercups (for
+            example, when a new message is created) at a URL of your choice.
```

Comment:
(should update the wiki soon with details about setting up and testing webhooks, and then add a link here)